themes/origins/sections/slideshow.liquid

```diff
@@ -0,0 +1,279 @@
+{{ 'component-slider.css' | asset_url | stylesheet_tag }}
+{{ 'component-slider.js' | asset_url | script_tag_deferred }}
+
+{% assign slideshow = section.settings %}
+
+<yc-section
+  {% if slideshow.type == 'full' %}
+    data-full
+  {% endif %}
+  style="
+    --section-space-top: {{ slideshow.space_top }}px;
+    --section-space-bottom: {{ slideshow.space_bottom }}px;
+  "
+>
+  <yc-slider
+    {% if slideshow.autoplay %}
+      autoplay
+    {% endif %}
```

Comment:
what do u think of this syntax? we’d need a minor change in attribute checking, from `hasAttr` to also confirming its `type` value. 
just a thought, not necessarily for this PR.

btw, here’s the settings data returned for arrows (checkbox type):
| Theme editor | Returned value |
|--------|--------|
| false | null |
| true | 1 | 

```suggestion
    autoplay={{ slideshow.autoplay }}
```

Reply:
both are nice, no right or wrong answer here. I do lean towards 
```
{% if slideshow.autoplay %}
      autoplay
    {% endif %}
```
just because it's the closest to the native attr behavior